chain-impl-mockchain/src/leadership/genesis/mod.rs

```diff
@@ -197,15 +198,132 @@ mod tests {
         (pool_id, pool_vrf_private_key)
     }
 
+    #[derive(Clone, Debug)]
+    pub struct LeaderElectionParameters {
+        slots_per_epoch: u32,
+        active_slots_coeff: f32,
+        pools_count: usize,
+        value: Value,
+    }
+
+    impl LeaderElectionParameters {
+        pub fn new() -> Self {
+            let pools_count = 5;
+            let active_slots_coeff = 0.18;
```

Comment:
why 0.18 ? why 5 pools ?

Reply:
Previously it was a property test with `pools_count` in range `5-10` and `active_slots_coeff` just to enable to each pool to be eligible to be chosen as leader (a little less than `1/pools_count`). But quickcheck test was a bit too long, so i chose some random set of values from some random run. Do you have different proposal?